docs/azure_arc_jumpstart/azure_arc_k8s/aks_hybrid/aks_edge_essentials/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: docs
```

Comment:
let's change the folder name _aks_edge_essentials_ >> __aks_edge_essentials_single_